pkg/proxy/iptables/proxier.go

```diff
@@ -273,6 +273,7 @@ func NewProxier(ipt utiliptables.Interface,
 		if err := sysctl.SetSysctl(sysctlRouteLocalnet, 1); err != nil {
 			return nil, fmt.Errorf("can't set sysctl %s: %v", sysctlRouteLocalnet, err)
 		}
+		klog.V(1).Info("Set sysctl %q to 1", sysctlRouteLocalnet)
```

Comment:
maybe just make `sysctl.SetSysctl` log this itself?

Reply:
I don't like logging from deep within libraries like that, but I can extract to proxy/util